### DISCUSSION TITLE
Information in alpha channel is lost when resizing a tiff

### DISCUSSION THREAD

**Discussion by gursch** [ImageMagick/ImageMagick#7614] [Help]
### ImageMagick version

7.1.1-38 Q16 x64 b0ab922:20240901

### Operating system, version and so on

Windows 11 Pro

### Description

Hi,

I have several TIFF files with an alpha channel. When I resize the files, the information in the alpha channel is gone. The alpha channel is becoming black.

For example: magick test.tif -resize 500x500 out.tif

Channels in Photoshop before the conversion:
![before](https://github.com/user-attachments/assets/e829a961-24f6-4204-bc84-59dc636c9ea2)

After the conversion:
![after](https://github.com/user-attachments/assets/e5430e7a-2352-414b-90df-1108525f5ca9)

Is there any way to preserve the alpha channels?

I prepared a test file (zipped): 
[test.zip](https://github.com/user-attachments/files/16989313/test.zip)

Thanks for your help
Alexander

**Comment by snibgo** [comment]
According to ImageMagick (IM) and exiftool, the image in your file test.tif has no alpha channel.

The image has 4 raster channels: RGB plus one other. It also contains a clipping path. The clipping path is vector, but is resized by "-resize 500x500".

exiftool calls the other raster channel "extra, unspecified". IM calls it a meta channel, "meta0". Gimp can read this extra channel as alpha, in the shape of the bananna.

Current IM thinks all values in that channel are zero. I think this is a bug. For example, using IM v7.1.1-38 (Beta) (LIBTIFF Version 4.4.0):
```
magick test.tif -channel meta0 ftxt:x.txt
```
`x.txt` has all values zero.

The same command with IM v7.1.1-20 Q16 (LIBTIFF Version 4.6.0) shows a variety of values, mostly 65535.

My conclusion: I think IM has a bug reading tiff "extra" channels. I don't know if the problem is in IM code or libtiff code.

**Comment by gursch** [comment]
Thanks for the analysis. I also think it's bug. I will create an issue.
